Bottom: bad72352ee9cd86d6e8954e57fb0e6f9dd163b01
Top:    fd778748f9f80d0ccf2edb497472465c04fcec88
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 22:09:10 -0500

Refresh of info

---

diff --git a/syntax.php b/syntax.php
index 7f938c0..b27eb91 100644
--- a/syntax.php
+++ b/syntax.php
@@ -16,13 +16,12 @@ class syntax_plugin_tablecalc extends DokuWiki_Syntax_Plugin {
      */
     function getInfo() {
         return array(
-                'author' => 'Gryaznov Sergey',
-                'email'  => 'stalker@os2.ru',
-                'date'   => '13-04-10',
-                'name'   => 'Table Calculations Plugin',
-                'desc'   => 'Enables Excel style formulas in table syntax',
-                'url'    => 'http://narezka.ru/tablecalc',
-                );
+            'author' => 'Christopher Voltz',
+            'email' => 'cjunk@voltz.ws',
+            'date' => '2016-04-29',
+            'name' => 'Table Calculations Plugin',
+            'desc' => 'Adds limited spreadsheet functionalit to tables',
+            'url' => 'https://github.com/cvoltz/tablecalc.git');
     }
 
     function getType() { return 'substition'; }
